### ISSUE TITLE
Tree shaking

### ISSUE THREAD

**Issue by lednhatkhanh** [phosphor-icons/phosphor-react#3]
The whole package will be included when I import any icons from phosphor-react.

Could you please consider supporting tree shaking? Thank you.

**Comment by rektdeckard** [comment]
@lednhatkhanh looks like a regression when we switched to using Parcel, I'll make sure it's fixed up in the next release!

**Comment by devrnt** [comment]
Seems like the ES module export is not discovered according to `bundlephobia`: https://bundlephobia.com/result?p=phosphor-icons@1.0.1

**Comment by rektdeckard** [comment]
@devrnt that's the vanilla JS lib you're inspecting, not the React lib

**Comment by devrnt** [comment]
@rektdeckard Oops my bad, thanks to point out

**Comment by devrnt** [comment]
Seems like same behaviour is occuring in a default nextjs project. (see README.md)

https://github.com/devrnt/nextjs-phosphor-react

* Branch [master](https://github.com/devrnt/nextjs-phosphor-react/blob/master/README.md) conains the size of `next build` without the use of `phosphor-react`.
* Branch [phosphor-react](https://github.com/devrnt/nextjs-phosphor-react/blob/phosphor-react/README.md)  has an import of the `Hand` component from `phosphor-react` in `pages/index.js`.

**TLDR:** First load JS size
* without use of `phosphor-react`: `65.1 kB`
* with `phosphor-react`: `285 kB` (even when just using one icon, all icons are included in bundle)

**Comment by devrnt** [comment]
Has been fixed for me in version `1.0.1`

**Comment by rektdeckard** [comment]
Yep, fixed up Rollup config in 78dec00. Glad to hear it's working for you!
